<?xml version="1.0"?>
<launch>
  <arg name="bagfile"/>
  <arg name="simulate" default="true"/>
  <arg name="group_name" default="manipulator" />
  <arg name="planning_time" default="10.0" />
  
  <node name="trajectory_replay" pkg="godel_surface_detection" type="trajectory_test_node" output="screen">
    <param name="bagfile_name" value="$(arg bagfile)"/>
    <param name="group_name" value="$(arg group_name)"/>
    <param name="planning_time" value="$(arg planning_time)" />
    <param name="simulate" value="$(arg simulate)"/>
  </node>
</launch>